exe/fcrepo_wrapper

```diff
@@ -8,7 +8,7 @@ options = {}
 OptionParser.new do |opts|
   opts.banner = "Usage: fcrepo_wrapper [options]"
 
-  opts.on("--config", "Load configuration from a file") do |v|
+  opts.on("-cFILE",  "--config FILE", "Load configuration from a file") do |v|
```

Comment:
Yes `-c` is used for cloud in solr_wrapper.
```
Usage: solr_wrapper [options]
...
        --config                     Load configuration from a file
...
    -c, --cloud                      Run solr in cloud mode
...
```

I don't have strong opinions.  I put `-c` in for consistency with the majority of the other fcrepo_wrapper commands that have short-name versions and hadn't really been thinking about solr_wrapper yet.

I'm good either way.  Let me know which you prefer.